src/conf/local-file.yaml

```diff
@@ -39,6 +39,8 @@ observability:
   system-observability: true
   network-observability: false
   write-logs-to-db: true
+  cron-job-time-for-purge-db: "10d0h0m00s"
+  purge-logs-from-db: true
```

Comment:
Do not bring this config(purge config) under observability. Bring this under generic config, so that in future it can be used for any DBs.